qiskit_qec/geometry/model/shell.py

```diff
@@ -100,9 +106,14 @@ def exclude(vertex_paths: List[List[Vertex]], qubit_data: QubitData) -> bool:
         """
         _debug = False
```

Comment:
Can we add this as parameter to this method? and switch print statements to `logger.debug` if it's truly a debug information, or `logger.info` otherwise?

Something like
```python
def extract(..., debug: bool = False):
   ...
   if debug:
       logger.debug("<MESSAGE>")
```

Reply:
Yes. This is the plan. @grace-harper-ibm and I talked about this yesterday.

Reply:
Done